transport/src/main/java/io/netty5/channel/ChannelOutboundInvoker.java

```diff
@@ -140,7 +140,21 @@ public interface ChannelOutboundInvoker extends FuturePromiseFactory {
      * handler can decide to continue reading.  If there's a pending read operation already, this method does nothing.
      * <p>
      * This will result in having the
-     * {@link ChannelHandler#read(ChannelHandlerContext)}
+     * {@link ChannelHandler#read(ChannelHandlerContext, ReadBufferAllocator)}
+     * method called of the next {@link ChannelHandler} contained in the {@link ChannelPipeline} of the
+     * {@link Channel}.
+     */
+    ChannelOutboundInvoker read(ReadBufferAllocator readBufferAllocator);
```

Comment:
do we need to describe expectations on combining/queuing behavior? for example if can handler wants to defer reading for some reason, and then what if another `read()` call comes before it is ready?

Reply:
Yeah I guess adding some more clarification cant hurt... let me think about how to word it.